string desire_type_name
string strategy_type_name
bool enabled
